math/median_test.go

```diff
@@ -0,0 +1,59 @@
+// author(s) [jo3zeph](https://github.com/jo3zeph)
+// median_test.go
+// see median.go
+
+package math
+
+import (
+	"testing"
+)
+
+func TestMedian(t *testing.T) {
+	testCases := []struct {
+		name       string
+		testValues []float64
+		ans        float64
+	}{
+
+		//test cases
```

Comment:
This comment would create issues

```suggestion
		// test cases
```

Reply:
But I actually remove removing this line completely, its clear what it is.

```suggestion
```